ee/codegen_integration/conftest.py

```diff
@@ -34,10 +36,29 @@ def _get_fixture_paths(root: str) -> Tuple[str, str]:
             "simple_inline_subworkflow_node",
             "simple_guardrail_node",
             "simple_map_node",
-            "simple_code_execution_node",
         }
     )
 )
 def code_to_display_fixture_paths(request) -> Tuple[str, str]:
     root = request.param
     return _get_fixture_paths(root)
+
+# Save the original `open`
+builtin_open = open
+
+# Fixture that mocks open for code execution nodes
+@pytest.fixture
+def mock_open_code_execution_file():
```

Comment:
Given that this fixture is only really use for code execution nodes, it'd be nice to colocate it closer to where it's used.

Reply:
Moved this to the bottom of the `test_code_to_display`